.github/workflows/qaqc_deploy.yml

```diff
@@ -4,26 +4,47 @@ on:
   workflow_dispatch:
 
 jobs:
-  deploy:
+  build_and_push:
     runs-on: ubuntu-22.04
     defaults:
       run:
         shell: bash
-      # working-directory: apps/qaqc ## currently needs to be run from root for git subtree command
-    container: 
-      image: dokku/ci-docker-image:latest
     env:
-      SSH_PRIVATE_KEY: ${{ secrets.DOKKU_SSH_KEY }}
-      DOKKU_HOST: ${{ secrets.DOKKU_HOST }}
-      APP_NAME: edm-data-engineering
-      GIT_REMOTE_URL: ssh://dokku@${{ secrets.DOKKU_HOST }}/edm-data-engineering
-      APP_REMOTE_BRANCH: main
-
+      DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+      DOCKER_USER: ${{ secrets.DOCKER_USER }}
     steps:
       - name: Checkout
         uses: actions/checkout@v3
         with:
-          sparse-checkout: apps/qaqc
-      - name: Deploy
-        run: apps/qaqc/bash/deploy.sh 
+          sparse-checkout: |
+            apps/qaqc
+            bash
+            dcpy
```

Comment:
just curious if a sparse checkout is still needed. we've definitely limited the idea to "QAQC app should use `dcpy`" so no worries, but maybe access to product-level files would be valuable in the future?